CI/check-jsonschema.py

```diff
@@ -0,0 +1,45 @@
+import json
```

Comment:
As I mentioned in the related PR (https://github.com/obsproject/obs-studio/pull/6360#discussion_r856933794) I'd prefer if this script uses a `main` method and also handles file access exceptions.